### ISSUE TITLE
Add dark theme

### ISSUE THREAD

**Issue by vitaly-zdanevich** [AntennaPod/antennapod.github.io#60]
We have a media query for that https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

I can do it.

**Comment by keunes** [comment]
If you have the time, drive & know-how to make a dark theme, please go ahead. Contributions for the site are very welcome :)

**Comment by ByteHamster** [comment]
I think building a dark color scheme is a pretty big task. One has to come up with a new palette that ensures proper contrast and then replace all graphics, too.

**Comment by vitaly-zdanevich** [comment]
I new to Jekyll, installed it according to [instruction](https://jekyllrb.com/docs/installation/) on Ubuntu 20.04.2, run `jekyll`, got:

```
Warning: the running version of Bundler (2.1.2) is older than the version that created the lockfile (2.1.4). We suggest you to upgrade to the version that created the lockfi
le by running `gem install bundler:2.1.4`.
Resolving dependencies...
Resolving dependencies...
Traceback (most recent call last):
21: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/ruby_executable_hooks:24:in `<main>'
20: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/ruby_executable_hooks:24:in `eval'
19: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/jekyll:23:in `<main>'
18: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/jekyll:23:in `load'
17: from /home/ubuntu/.rvm/gems/ruby-2.7.0/gems/jekyll-4.2.0/exe/jekyll:11:in `<top (required)>'
16: from /home/ubuntu/.rvm/gems/ruby-2.7.0/gems/jekyll-4.2.0/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
15: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler.rb:149:in `setup'
14: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/runtime.rb:20:in `setup'
13: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/runtime.rb:101:in `block in definition_method'
12: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:226:in `requested_specs'
11: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:237:in `specs_for'
10: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:170:in `specs'
 9: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:258:in `resolve'
 8: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/resolver.rb:22:in `resolve'
 7: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/resolver.rb:50:in `start'
 6: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolver.rb:43:in `resolve'
 5: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolution.rb:182:in `resolve'
 4: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolution.rb:257:in `process_topmost_state'
 3: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolution.rb:308:in `unwind_for_conflict'
 2: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolution.rb:308:in `tap'
 1: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolution.rb:310:in `block in unwind_for_conflict'
/home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/vendor/molinillo/lib/molinillo/resolution.rb:328:in `raise_error_unless_state': Unable to satisfy the following re
quirements: (Bundler::Molinillo::VersionConflict)

- `html-pipeline (~> 2.3)` required by `jekyll-mentions (1.6.0)`
15: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/ruby_executable_hooks:24:in `<main>'
14: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/ruby_executable_hooks:24:in `eval'
13: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/jekyll:23:in `<main>'
12: from /home/ubuntu/.rvm/gems/ruby-2.7.0/bin/jekyll:23:in `load'
11: from /home/ubuntu/.rvm/gems/ruby-2.7.0/gems/jekyll-4.2.0/exe/jekyll:11:in `<top (required)>'
10: from /home/ubuntu/.rvm/gems/ruby-2.7.0/gems/jekyll-4.2.0/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
 9: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler.rb:149:in `setup'
 8: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/runtime.rb:20:in `setup'
 7: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/runtime.rb:101:in `block in definition_method'
 6: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:226:in `requested_specs'
 5: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:237:in `specs_for'
 4: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:170:in `specs'
 3: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/definition.rb:258:in `resolve'
 2: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/resolver.rb:22:in `resolve'
 1: from /home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/resolver.rb:45:in `start'
/home/ubuntu/.rvm/rubies/ruby-2.7.0/lib/ruby/2.7.0/bundler/resolver.rb:59:in `rescue in start': Bundler could not find compatible versions for gem "html-pipeline": (Bundler:
:VersionConflict)
  In Gemfile:
    jekyll-mentions was resolved to 1.6.0, which depends on
      html-pipeline (~> 2.3)

Could not find gem 'html-pipeline (~> 2.3)', which is required by gem 'jekyll-mentions', in any of the sources.
```
Maybe switch to Hugo? )

**Comment by ByteHamster** [comment]
How about `gem install html-pipeline -v 2.3`?

> Maybe switch to Hugo?

No, sorry. Just because there is one build problem on one computer doesn't mean that the whole website should be rewritten.
